cmd/image.go

```diff
@@ -0,0 +1,112 @@
+package cmd
+
+import (
+	"os"
+	"strings"
+
+	"github.com/massdriver-cloud/massdriver-cli/pkg/api2"
+	"github.com/massdriver-cloud/massdriver-cli/pkg/image"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+type PushFlag struct {
+	Flag      string
+	Attribute *string
+}
+
+var imageCmd = &cobra.Command{
+	Use:   "image",
+	Short: "Container image integration Massdriver",
+	Long:  ``,
+}
+
+var imagePushCmd = &cobra.Command{
+	Use:   "push <namespace>/<image-name>",
+	Short: "Push an image to ECR, ACR or GAR",
+	RunE:  runImagePush,
+	Args:  cobra.ExactArgs(1),
+}
+
+var DockerBuildContext string
+var DockerfileName string
+var Tag string
+var ArtifactId string
+var ImageName string
+var Region string
+
+func init() {
+	rootCmd.AddCommand(imageCmd)
+	imageCmd.AddCommand(imagePushCmd)
+	imagePushCmd.Flags().StringVarP(&DockerBuildContext, "build-context", "b", ".", "Path to the directory to build the image from")
+	imagePushCmd.Flags().StringVarP(&DockerfileName, "dockerfile", "f", "Dockerfile", "Name of the dockerfile to build from if you have named it anything other than Dockerfile")
+	imagePushCmd.Flags().StringVarP(&Tag, "image-tag", "t", "latest", "Unique identifier for this version of the image")
+	imagePushCmd.Flags().StringVarP(&ArtifactId, "artifact", "a", "", "Massdriver ID of the artifact used to create the repository and generate repository credentials.")
+	imagePushCmd.MarkFlagRequired("artifact")
+	imagePushCmd.Flags().StringVarP(&Region, "region", "r", "", "Cloud region to push the image to")
+	imagePushCmd.MarkFlagRequired("region")
+}
+
+func runImagePush(cmd *cobra.Command, args []string) error {
+	setupLogging(cmd)
+
+	orgID := os.Getenv("MASSDRIVER_ORG_ID")
+	if orgID == "" {
+		log.Fatal().Msg("MASSDRIVER_ORG_ID must be set")
+	}
+
+	APIKey := os.Getenv("MASSDRIVER_API_KEY")
+	if orgID == "" {
```

Comment:
```suggestion
	if APIKey == "" {
```
Looks like this was a copy pasta error